composer.lock

```diff
@@ -161,36 +161,37 @@
             ],
             "description": "Symfony Console Component",
             "homepage": "https://symfony.com",
-            "time": "2019-12-01T10:04:45+00:00"
+            "time": "2020-09-09T05:09:37+00:00"
         },
         {
             "name": "symfony/debug",
-            "version": "v3.4.36",
+            "version": "v4.4.15",
```

Comment:
Kinda weird that this was upgraded to 4.4  O.o

Reply:
Is that a problem for this here? Or do you see a fix?

Reply:
well, phpBB uses symfony 3.4, and you should not mix them in general.

Reply:
Wow that is seriously weird. I can't see why composer would try and get v4 of anything when we've marked the components as v3 in composer.json. I just tried doing a composer install as well and the same thing happened to me.

All I can think of is we are running newer versions of PHP locally (I'm on PHP 7.1.33) and maybe the old debug component only runs on older versions? Maybe on .com it would get the v3 component?

```vagrant@ubuntu-bionic:/var/www/html/phpbb-translation-validator$ composer show -                                                                                                                                                             -tree
battye/php-array-parser v1.0.8 Parse text representation of a PHP array into an                                                                                                                                                              actual PHP array.
└──php >=5.3.3
phpunit/phpunit 5.7.27 The PHP Unit Testing framework.
├──ext-dom *
├──ext-json *
├──ext-libxml *
├──ext-mbstring *
├──ext-xml *
├──myclabs/deep-copy ~1.3
│  └──php ^7.1 || ^8.0
├──php ^5.6 || ^7.0
├──phpspec/prophecy ^1.6.2
│  ├──doctrine/instantiator ^1.0.2
│  │  └──php ^7.1 || ^8.0
│  ├──php ^5.3|^7.0
│  ├──phpdocumentor/reflection-docblock ^2.0|^3.0.2|^4.0|^5.0
│  │  ├──php ^7.0
│  │  ├──phpdocumentor/reflection-common ^1.0.0 || ^2.0.0
│  │  │  └──php >=7.1
│  │  ├──phpdocumentor/type-resolver ~0.4 || ^1.0.0
│  │  │  ├──php ^7.1
│  │  │  └──phpdocumentor/reflection-common ^2.0
│  │  │     └──php >=7.1
│  │  └──webmozart/assert ^1.0
│  │     ├──php ^5.3.3 || ^7.0 || ^8.0
│  │     └──symfony/polyfill-ctype ^1.8
│  │        └──php >=7.1
│  ├──sebastian/comparator ^1.2.3|^2.0|^3.0|^4.0
│  │  ├──php >=5.3.3
│  │  ├──sebastian/diff ~1.2
│  │  │  └──php ^5.3.3 || ^7.0
│  │  └──sebastian/exporter ~1.2 || ~2.0
│  │     ├──php >=5.3.3
│  │     └──sebastian/recursion-context ~2.0
│  │        └──php >=5.3.3
│  └──sebastian/recursion-context ^1.0|^2.0|^3.0|^4.0
│     └──php >=5.3.3
├──phpunit/php-code-coverage ^4.0.4
│  ├──ext-dom *
│  ├──ext-xmlwriter *
│  ├──php ^5.6 || ^7.0
│  ├──phpunit/php-file-iterator ^1.3
│  │  └──php >=5.3.3
│  ├──phpunit/php-text-template ^1.2
│  │  └──php >=5.3.3
│  ├──phpunit/php-token-stream ^1.4.2 || ^2.0
│  │  ├──ext-tokenizer *
│  │  └──php ^7.0
│  ├──sebastian/code-unit-reverse-lookup ^1.0
│  │  └──php ^5.6 || ^7.0
│  ├──sebastian/environment ^1.3.2 || ^2.0
│  │  └──php ^5.6 || ^7.0
│  └──sebastian/version ^1.0 || ^2.0
│     └──php >=5.6
├──phpunit/php-file-iterator ~1.4
│  └──php >=5.3.3
├──phpunit/php-text-template ~1.2
│  └──php >=5.3.3
├──phpunit/php-timer ^1.0.6
│  └──php ^5.3.3 || ^7.0
├──phpunit/phpunit-mock-objects ^3.2
│  ├──doctrine/instantiator ^1.0.2
│  │  └──php ^7.1 || ^8.0
│  ├──php ^5.6 || ^7.0
│  ├──phpunit/php-text-template ^1.2
│  │  └──php >=5.3.3
│  └──sebastian/exporter ^1.2 || ^2.0
│     ├──php >=5.3.3
│     └──sebastian/recursion-context ~2.0
│        └──php >=5.3.3
├──sebastian/comparator ^1.2.4
│  ├──php >=5.3.3
│  ├──sebastian/diff ~1.2
│  │  └──php ^5.3.3 || ^7.0
│  └──sebastian/exporter ~1.2 || ~2.0
│     ├──php >=5.3.3
│     └──sebastian/recursion-context ~2.0
│        └──php >=5.3.3
├──sebastian/diff ^1.4.3
│  └──php ^5.3.3 || ^7.0
├──sebastian/environment ^1.3.4 || ^2.0
│  └──php ^5.6 || ^7.0
├──sebastian/exporter ~2.0
│  ├──php >=5.3.3
│  └──sebastian/recursion-context ~2.0
│     └──php >=5.3.3
├──sebastian/global-state ^1.1
│  └──php >=5.3.3
├──sebastian/object-enumerator ~2.0
│  ├──php >=5.6
│  └──sebastian/recursion-context ~2.0
│     └──php >=5.3.3
├──sebastian/resource-operations ~1.0
│  └──php >=5.6.0
├──sebastian/version ^1.0.6|^2.0.1
│  └──php >=5.6
└──symfony/yaml ~2.1|~3.0|~4.0
   ├──php ^5.5.9|>=7.0.8
   └──symfony/polyfill-ctype ~1.8
      └──php >=7.1
symfony/console v3.4.46 Symfony Console Component
├──php ^5.5.9|>=7.0.8
├──symfony/debug ~2.8|~3.0|~4.0
│  ├──php >=7.1.3
│  ├──psr/log ~1.0
│  │  └──php >=5.3.0
│  └──symfony/polyfill-php80 ^1.15
│     └──php >=7.1
└──symfony/polyfill-mbstring ~1.0
   └──php >=7.1
symfony/finder v3.4.46 Symfony Finder Component
└──php ^5.5.9|>=7.0.8
symfony/yaml v3.4.46 Symfony Yaml Component
├──php ^5.5.9|>=7.0.8
└──symfony/polyfill-ctype ~1.8
   └──php >=7.1
```

Reply:
@Crizz0 maybe the composer.lock change can be removed from the PR so that only the `LangKeyValidator.php` code change flows through?

Reply:
The problem for us would be when phpbb-translation-validator gets pulled into Titania on the website - I think then the Symfony package conflicts would cause us a problem like paul999 said